client/src/containers/MainPage/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from "react";
+import { Box } from "@material-ui/core";
```

Comment:
remove some of these imported modules if unused

Reply:
Removed unused imported modules